app/views/user_sessions/new.html.erb

```diff
@@ -22,7 +22,8 @@
 
         <div class="flex flex-col items-center mb-10">
           <%= link_to "Forgot password?", new_password_reset_path, class: "font-black italic underline text-white mb-10" %>
-          <%= link_to "Sign up", new_registration_path, class: "font-black italic underline text-white" %>
+          <%= link_to "Sign up", new_registration_path, class: "font-black italic underline text-white mb-10" %>
+          <%= link_to "Continue as guest", sessions_path, class: "font-black italic underline text-white" %>
```

Comment:
I am waiting for Charly's opinion on the design.

Reply:
I updated the PR based on this design:
![image](https://github.com/user-attachments/assets/ed232e05-c3b8-4cad-b28f-b9507e592255)